api/productions.js

```diff
@@ -1,3 +1,7 @@
+/* eslint-disable no-console */
+import MockSuggestedEvents from '../assets/suggested-events';
+const isDevelopmentEnviroment = process.env.NODE_ENV === 'development';
```

Comment:
All `process.env.` variables are hardcoded during a build.

So usually, we should use the `publicRuntimeConfig` in `nuxt.config.js` instead. We can access those variables through the Nuxt context in plugins, components, middleware, etc.

If you cannot access the Nuxt context because your code is a vanilla JS function outside of Nuxt, you'll need to pass the config you need as arguments to those functions. An example where we do this is here: https://github.com/cultuurnet/udb3-frontend/blob/main/plugins/api.js#L4-L10

See also the official documentation: https://nuxtjs.org/blog/moving-from-nuxtjs-dotenv-to-runtime-config/

> At runtime, Node.js automatically loads environment variables into process.env so that they are available to use in your application. The reference to the environment variable is replaced with the correct value. For example, if you had an API_SECRET key with the value of 'my-secret' then in your application where you had used process.env.API_SECRET this would be replaced with the value of my-secret.
> 
> __Values are read during build time and persisted in the webpack bundle. Therefore if we change our API_SECRET we will need to rebuild our application so that it can read the new value.__

(Emphasis mine)

**However in this case of `process.env.NODE_ENV` I think it would be fine because the variable is either `dev` if we're running `yarn dev`, and `production` when we run `yarn build` to build it to run on the servers.** In other words, the variable indicates what kind of build you're running and thus requires a re-build anyway to change.

However there is one other issue: If we run `yarn build` locally to run the app in production mode in our local Nuxt server in the Vagrant box, it will also be running as `process.env.NODE_ENV === production`. So it would still try to fetch actual suggestions, while we'd probably want to fetch the mock suggestions even though we're running in `production` mode.

So I would still advise putting a custom `environment` variable in nuxt.config.js instead, so we can set that to `development` for our local installs (both with `yarn dev` and `yarn build`), and also have more options like `acceptance`, `test`, and `production` instead of just `development` & `production`.

(cc @geoffreydhuyvetters because I think you might have missed this change to how we deal with environment variables, we spotted this while setting up the build pipeline while you were on vacation)